LinearApproach/LinearApproachMultiCCA.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def iterative_process_K(datasets:list, penalties:list, K:int):
</pre><p>Rename to something like <code>lp_pmd</code> o distinguish it from <code>multicca_pmd</code>  and then make sure it has matching return types. Also update the docstrings to match.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/sparsecca/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>updated:) Hope it has the correct type?</p><p> </p><p> </p><p> </p>